design-system/materials/resets.css

```diff
@@ -0,0 +1,41 @@
+*,
+*::before,
+*::after {
+  box-sizing: inherit;
+}
+
+html,
+body {
+  color: var(--font-color-for-text);
+  font-family: var(--font-family);
+  margin: 0;
+  padding: 0;
+}
+
+html {
+  box-sizing: border-box;
+}
+
+h1,
+h2,
+h3,
+h4,
+h5,
+h6 {
+  margin: 0;
+}
+
+button {
+  cursor: pointer;
+}
+
+a {
+  color: var(--color-primary);
+  cursor: pointer;
+  text-decoration: none;
+  transition: color 0.2s ease-in;
+}
+
+p {
+  margin: 0;
+}
```

Comment:
Small change! The paragraph should not have the `color` style assigned as it conflicts with disabled styles.

Reply:
Could be although it seemed an edge case according to the VRTs: the `LocalizedRichTextInput` in read-only mode.

See https://percy.io/commercetools-GmbH/ui-kit/builds/22876206/changed/1276880837?browser=chrome&browser_ids=31&subcategories=unreviewed%2Cchanges_requested&viewLayout=side-by-side&viewMode=new&width=1024&widths=1024

Do you have other suggestions?

Reply:
Ok, so then this is an actual bug in `app-kit` (I checked it locally) and this change would fix it.

Thanks.